### ISSUE TITLE
Submission to conda-forge

### ISSUE THREAD

**Issue by andersy005** [andersy005/conda-recipe-cudatoolkit-dev#7]
@scopatz, @xmnlab, 
I've just finished the needed work for Linux support and I ran tests on both my local computer and an AWS instance. 
Most things seem to be working perfectly! I was wondering whether this would be a good time to submit the package to conda-forge or wait until Windows and Osx are supported?

Thanks!

**Comment by xmnlab** [comment]
Sorry I saw this repo just now .. maybe would be better to add this recipe to https://github.com/Quansight/conda-recipes/

and you can create a branch cudatoolkit-dev there

**Comment by xmnlab** [comment]
about to submit to conda-forge .. I think it would be the PR .. you can add work-in-progress label [WIP] int the title of you PR.

**Comment by andersy005** [comment]
@xmnlab, 
I created this repo for the recipe as independent from other recipes so that:
- I can easily add CIs.. 
- It is so easy to keep track of the discussions about the cudatoolkit-dev package 

Once most of the work is finished, I can move/copy the final work (recipe) to the Quansight recipes as well.

**Comment by xmnlab** [comment]
OK. maybe would be good to start to think about create a CI for conda-recipes repo ... and write some guidelines. I already something in mind .. I will create a issue there about this.

**Comment by scopatz** [comment]
Maybe we should just have a fork of conda-forge/staged-recipes

**Comment by xmnlab** [comment]
@scopatz  we have already one: https://github.com/Quansight/staged-recipes
I agree, for recipes that we want to make a PR to conda-forge would be better to work on this repo.

for packages we want to upload to anaconda.org I think it would be better to keep them on our conda-recipes (https://github.com/Quansight/conda-recipes/).

what do you'll think about that?

**Comment by scopatz** [comment]
@xmnlab - yeah that is true.  I think we mostly should want to go through conda-forge, though :)

**Comment by xmnlab** [comment]
@scopatz you're right .. we just decided to upload to quansight account some packages that maybe will moved to the client account or some packages that we need some extra features that conda-forge didn't support.
